src/components/shared/Footer.js

```diff
@@ -4,17 +4,17 @@ import s from 'styled-components'
 import { FUTURA_REGULAR } from '../../utils/font'
 
 const Credit = s.div`
-  color: #000000;
+  color: ${({ fontColor = '#000000' }) => fontColor};
   ${({ font = FUTURA_REGULAR }) => font}
   padding: 1rem;
   font-size: 80%;
   text-align: center;
   background-color: ${({ bgColor = '#FFFFFF' }) => bgColor};
 `
 
-export const Footer = ({ font, emoji, bgColor }) => (
-  <Credit font={font} bgColor={bgColor}>
+export const Footer = ({ font, emoji, bgColor, fontColor, year }) => (
```

Comment:
you can use a default value here instead of casing on the truthy value of year I think